connect/runtime/src/test/java/org/apache/kafka/connect/integration/MonitorableSinkConnector.java

```diff
@@ -32,7 +32,9 @@
 import java.util.Map;
 
 /**
- * A connector to be used in integration tests. This class provides methods to find task instances
```

Comment:
Please skip this file and review: https://github.com/apache/kafka/pull/6342 instead. Thanks!

Reply:
#6342 was merged. This PR is now rebased on top of it.